puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/pulpcore_packaging_release.groovy

```diff
@@ -0,0 +1,74 @@
+def packages_to_build
```

Comment:
this file should not be here but folded with `foreman_packaging_release.groovy`, but I was too lazy to parametrize it properly